Source/Core/Core/HW/EXI_DeviceEthernet.cpp

```diff
@@ -358,6 +358,12 @@ void CEXIETHERNET::MXCommandHandler(u32 data, u32 size)
 		data &= (data & 0xff) ^ 0xff;
 		goto write_to_register;
 
+	case BBA_TXFIFOCNT:
```

Comment:
I was considering this, but that would mean a check every frame instead of once. Libogc will always write there in 2 writes. And commercial games never write to FIFOCNT at all.